run_build.sh

```diff
@@ -18,15 +51,24 @@ git submodule init
 git submodule update
 git merge --no-edit origin/tanabarr/control-no-ipmctl-May2020
 git merge --no-edit origin/mjmac/allow-fwd-disable-20200508
-scons --build-deps=yes --config=force install
+scons MPI_PKG=any --build-deps=yes --config=force install
```

Comment:
I think it might be worth checking for daos build success here. 
Can we check for scons return code to ensure success, instead of checking for binaries existence?
If daos build fails for any reason, bail here, do not build ior since it will likely not link to the correct daos library.

Reply:
I just added a trap and set -e so, whenever something on the run_build.sh script crashes. It will stop its execution and will report the error to the console. I prefer to still continue checking that the binaries actually exist. Otherwise, if scons silently fails. We will continue launching all the slurm jobs that will end up failing. I recall that two weeks ago or so. Folks reported that the daos binary was missing in their builds